### DISCUSSION TITLE
Label mnemonic not working

### DISCUSSION THREAD

**Discussion by TangDave** [AvaloniaUI/Avalonia#16089] [Q&A] [answered]
Using a mnemonic on a label should cause focus to be set to the control next in the tab order, but it doesn't appear to be working.
The label accepts the mnemonic fine - on pressing the 'alt' key the mnemonic shows up on the label, but doesn't ever actually do anything, at least when the label is followed by a textbox.

To reproduce just set a label with Content such as "_foo" and follow by a textbox. At runtime 'alt-f' should set focus to the textbox.

**Comment by rabbitism** [comment, marked as answer]
Did you set label's Target?

**Reply by TangDave** [reply]
Beautiful! I'm coming from a WinForms background and there it's assumed to be the next control in the tab order, but this is more elegant. Thanks.

**Reply by TangDave** [reply]
Works for TextBox target, but not for ComboBox target - is this by design?
